color.js

```diff
@@ -12,11 +12,23 @@ export class Color {
   }
 
   //"Factory method" that creates color from given HEX value
-  static fromHex(hexColor) {
+  static hexToRGB(hexColor) {
+    const r = parseInt(hexColor.slice(1, 3), 16);
+    const g = parseInt(hexColor.slice(3, 5), 16);
+    const b = parseInt(hexColor.slice(5, 7), 16);
+    const res = { r, g, b };
```

Comment:
Here it would be better to create an instance of Color class with `new Color(r, g, b);`